Docs (#4)
yiisoft/psr-emitter: docs/internals.md

./vendor/bin/roave-infection-static-analysis-plugin
```

> Running mutation testing requires disabling the same functions as unit testing. Do it by `php.ini` way.

Reviewer: ```suggestion
Running mutation testing requires disabling the same functions as unit testing. Do it in `php.ini`.
```
